helm/kong-app/CHANGELOG.md

```diff
@@ -14,7 +73,7 @@ hybrid mode.
 
 ### Improvements
 
-* Update default Kong version to 2.1.3
+* Update default Kong version to 2.1.
```

Comment:
this is the only bit i'm unsure about - the chart appears to use 2.1.3. was this an accidental deletion?